code/datums/supplypacks.dm

```diff
@@ -848,6 +848,21 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 /datum/supply_packs/security_formal_wear/New()
 	selection_from = list(Blue, Navy, Tan)
 
+/datum/supply_packs/nfl_suits
+	name = "NFL Suit and training dummy"
+	contains = list(/obj/structure/mannequin,
+					/obj/item/clothing/head/syndie_football_helmet,
+					/obj/item/clothing/head/syndie_football_helmet,
+					/obj/item/clothing/head/syndie_football_helmet,
+					/obj/clothing/suit/syndie_football,
+					/obj/clothing/suit/syndie_football,
```

Comment:
Bruh, why. Just add the syndie football clothing to the contraband tab, so you keep consistent with the team security vendor, which has team geometer (cult) gear under contraband.

Reply:
Its thr training dummy so you give it to train on the dummy